plugin.cfg

```diff
@@ -64,3 +64,4 @@ erratic:erratic
 whoami:whoami
 on:github.com/coredns/caddy/onevent
 sign:sign
+minimal:minimal
```

Comment:
We'd want to cache minimal responses (so it needs to be after cache), but minimal will want to minimize all responses created by other plugins (so it needs to go before them). 

So, i'm thinking right before template, the first plugin in the chain that creates responses.

Reply:
My thinking was that it should be the last plugin to be processed since it is potentially stripping off certain records from the response. Caching makes sense here. 

Ahh I think I missed out the fact that minimal is wrapping up a response writer. This makes sense! Thank you for the insights!